CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] Query Frontend: depcrated `-querier.compress-http-responses-deprecated`. You should use `-api.response-compression-enabled` instead. #3544
```

Comment:
```suggestion
* [CHANGE] Query Frontend: deprecate `-querier.compress-http-responses-deprecated` in favor of `-api.response-compression-enabled`. #3544
```

Reply:
The flag appears to be named `-querier.compress-http-responses` ?